reactioncommerce/reaction-admin
# Refactor localization settings form

Resolves #226 
Impact: **minor**
Type: **refactor|chore**

## Summary
De-meteorize localization settings form. 

**Corresponding API PR**: https://github.com/reactioncommerce/reaction/pull/6146

## Solution
Refactor localization form to use reacto-form

## Testing
1. Verify that all fields in the new localization form update correctly.
2. Verify that when UOL/UOM are updated, the default parcel size is updated accordingly
